en: update Grafana panels and add relay log notes (#252)
pingcap/docs-dm: en/monitor-a-dm-cluster.md

| binlog file gap between master and syncer | The number of binlog files by which the binlog replication unit is behind the upstream master | N/A | N/A |
| shard lock resolving | Whether the current subtask is waiting for sharding DDL replication. A value greater than 0 means that the current subtask is waiting for sharding DDL replication | N/A | N/A |

### Operation errors

| Metric name | Description | Alert | Severity level |
|:----|:------------|:----|:----|
| before any operate error | The number of errors before any operation | N/A | N/A |

> | source bound error | The number of data source binding operations | N/A | N/A |

Reviewer: ```suggestion
| source bound error | The number of errors of data source binding operations | N/A | N/A |
```

> | source bound error | The number of data source binding operations | N/A | N/A |

Reviewer 2: ```suggestion
| source bound error | The number of data source binding operation errors | N/A | N/A |
```

Author: Thanks! I'll choose the version suggested by @TomShawn, which might be easier to read.

| start error | The number of errors during the start of a subtask | N/A | N/A |
| pause error | The number of errors during the pause of a subtask | N/A | N/A |
| resume error | The number of errors during the resuming of a subtask | N/A | N/A |
| auto-resume error | The number of errors during the auto-resuming of a subtask | N/A | N/A |
| update error | The number of errors during the update of a subtask | N/A | N/A |
| stop error | The number of errors during the stop of a subtask | N/A | N/A |

### High availability

| Metric name | Description | Alert | Severity level |
|:----|:------------|:----|:----|
| number of dm-masters start leader components per minute | The number of DM-master attempts to enable leader related components per minute | N/A | N/A |
| number of workers in different state | The number of DM-workers in different states | Some DM-worker(s) has (have) been offline for more than one hour | critical |
| workers' state | The state of the DM-worker | N/A | N/A |
| number of worker event error | The number of different types of DM-worker errors | N/A | N/A |

> | shard ddl error per minute | The number of different types of sharding DDL errors per minute | A sharding DDL error occurs | critical |

Reviewer 2: how about
```suggestion
| shard ddl error per minute | The number of different types of sharding DDL errors per minute | Any sharding DDL error occurs | critical |
```

> | number of pending shard ddl | The number of pending sharding DDL operations | The number of pending sharding DDL operations has existed for more than one hour | critical |

Reviewer 2: ```suggestion
| number of pending shard ddl | The number of pending sharding DDL operations | Pending sharding DDL operations has existed for more than one hour | critical |
```

| unsynced tables | The number of tables that have not received the shard DDL statement in the current subtask | N/A | N/A |
| shard lock resolving | Whether the current subtask is waiting for the shard DDL lock to be resolved. A value greater than 0 indicates that it is waiting for the shard DDL lock to be resolved | N/A | N/A |

### Relay log

> **Note:**
>
> Currently, DM v2.0 does not support enabling the relay log feature.

> | Metric name | Description | Alert | Severity level |

Author: @TomShawn You don't need to review this table because it's moved from a place above in this file and no changes are made.

Reviewer: Got it. Thanks!
